DeAceroDLabs/marketplace-frontend
# Feature/dmp 508

## What is done

[[DMP-508] Shopping cart items not found page](https://deacero.atlassian.net/browse/DMP-432)

## How is done

- Created a `ShoppingCartItemsNotFound` component that will be used when the user deletes all products from the ShoppingCart view.
- Changed `Cart`  name to `ShoppingCart`.
- Changed  `CartItem`  name to `ShoppingCartItem`.
- Added `UpperTitle` prop to `NotFound` component.

## How to test

1. Run the project via `npm start`
2. Login.
3. Go to /carrito/productos.
4. Delete all the products, you should see the `ShoppingCartItemsNotFound` view.
